rootfs/Dockerfile

```diff
@@ -9,28 +9,18 @@ RUN adduser --system \
 
 COPY . /app
 
-RUN apt-get update \
-	&& apt-get install -y \
-		gcc \
-		git \
-		libffi-dev \
-		libpq5 \
-		libpq-dev \
-		python3 \
-		python3-dev \
-		sudo \
-	&& ln -s /usr/bin/python3 /usr/bin/python \
-	&& curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.1 \
-	&& mkdir -p /configs && chown -R deis:deis /configs \
-	&& pip install --disable-pip-version-check --no-cache-dir -r /app/requirements.txt \
-	&& apt-get remove -y --auto-remove --purge \
-		gcc \
-		git \
-		libffi-dev \
-		libpq-dev \
-		python3-dev \
-	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc
+RUN buildDeps='gcc git libffi-dev libpq-dev libpq-dev python3-dev'; \
+	# disable source repos (speeds up apt-get update)
+	sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list && \
+	apt-get update && \
+	apt-get install -y $buildDeps python3 sudo --no-install-recommends && \
+	ln -s /usr/bin/python3 /usr/bin/python && \
+	curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.1 && \
```

Comment:
Yeah, but that is not what this PR is about (pip that is). 

8.1.2 has been the current for a long time and we haven't updated, we can do that as a separate PR.